extensions/quarkiverse-kerberos.yaml

```diff
@@ -3,8 +3,17 @@ group-id: "io.quarkiverse.kerberos"
 artifact-id: "quarkus-kerberos"
 versions:
 - "1.0.0"
```

Comment:
```suggestion
- "1.0.0"
```
```suggestion
- "1.0.0":
```

Reply:
The `:` is missing next to the versions (YAML sucks)

Reply:
Thanks for spotting it, I just was not careful enough :-)